### PR TITLE
Created Item Classes and its related components and in midst of implementing ItemTest and ItemNameTest

### LINKED ISSUE #85: Create Item objects and it's associated classes (Eg: ItemPrice and ItemName)

### PR DESCRIPTION
**Pull request by WillySeahh**

### CODE REVIEW

File: docs/AboutUs.adoc [hunk 1]
```diff
@@ -13,9 +13,10 @@ We are a team based in the http://www.comp.nus.edu.sg[School of Computing, Natio
 
 === Willy Seah
 image::willyseahh.png[width="150", align="left"]
-{empty}[http://www.comp.nus.edu.sg/WillySeahh[homepage]] [https://github.com/WillySeahh[github]] [<<WillySeahh#, portfolio>>]
+{empty}[https://github.com/WillySeahh[github]] [<<willyseahh#, portfolio>>]
 
-Role: Project Advisor
+Role: Project Advisor +
+Responsibilities: Team Facilitation, Code Integration, Expense Splitter
```
Comment (sheexiong) [inline docs/AboutUs.adoc:19]: @akhilvuputuri @Jarrod-Bob @ailanthustng we should also go update our role and responsibilities asap.

File: src/main/java/seedu/address/model/item/Item.java [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+     * Every field must be present,
+     * ItemName and ItemPrice fields must not be null.
+     */
+    public Item(ItemName name, ItemPrice price) {
+        requireAllNonNull(name, price);
+        this.name = name;
+        this.price = price;
+    }
+
+    public ItemName getName() {
```
Comment (sheexiong) [inline src/main/java/seedu/address/model/item/Item.java]: Maybe you should change method name to `getItemName` to provide a better description for the method.

File: src/main/java/seedu/address/model/item/Item.java [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        return name;
+    }
+
+    public ItemPrice getPrice() {
```
Comment (sheexiong) [inline src/main/java/seedu/address/model/item/Item.java]: Same as the `getName` above, should name as `getItemPrice` instead.

File: src/main/java/seedu/address/model/item/ItemName.java [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+     */
+    public ItemName(String name) {
+        requireNonNull(name);
+        checkArgument(isValidName(name), MESSAGE_CONSTRAINTS);
+        this.itemName = name;
+    }
+
+    /**
+     * Returns true if a given string is a valid name.
+     */
```
Comment (sheexiong) [inline src/main/java/seedu/address/model/item/ItemName.java]: Maybe you could provide the parameter in the javadoc comment.

**Review by sheexiong — COMMENTED** [review]
I think overall coding is okay, but one of the suggestions I would give is could include the parameter field in the javadoc for the method.

File: src/main/java/seedu/address/model/item/Item.java (hunk 2 above)
Comment (WillySeahh) [reply src/main/java/seedu/address/model/item/Item.java]: Alright understood. thank you

**Review by sheexiong — APPROVED** [review]
LGTM

**Pull request merged**